jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ClientConnector.java

```diff
@@ -753,4 +768,29 @@ public ChannelWithAddress newChannelWithAddress(ClientConnector clientConnector,
             };
         }
     }
+
+    public interface ConnectListener extends EventListener
```

Comment:
Please add javadocs to class and methods.

Reply:
Of course, I will do that as soon as API is hardened.